config/migrations/2023/20231106111659-cleanup-decisions-types.sparql

```diff
@@ -0,0 +1,69 @@
+PREFIX BesluitType:                      <https://data.vlaanderen.be/id/concept/BesluitType/>
+PREFIX BesluitDocumentType:              <https://data.vlaanderen.be/id/concept/BesluitDocumentType/>
+PREFIX skos:                             <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd:                              <http://www.w3.org/2001/XMLSchema#>
+PREFIX besluit:                          <http://lblod.data.gift/vocabularies/besluit/>
+PREFIX BestuurseenheidClassificatieCode: <http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/>
+PREFIX lblodRule:                        <http://data.lblod.info/id/notification-rule/>
+
+# Delete duplicate obligationToReport Booleans
+DELETE {
+    GRAPH ?g {
+        lblodRule:1f21f0df-4956-4665-9889-f2f1ea1824be besluit:obligationToReport "0"^^xsd:boolean .
+    }
+} 
+WHERE {
+    GRAPH ?g {
+        lblodRule:1f21f0df-4956-4665-9889-f2f1ea1824be besluit:obligationToReport "0"^^xsd:boolean .
+    }
+}
+
+;
+
+# Delete the decidableBy "Autonoom gemeentebedrijf" from lblodRule which has obligationToReport equal to false
+DELETE {
+    GRAPH ?g {
+        lblodRule:7fde1283-f580-4452-a5c4-f2234990d6e9 besluit:decidableBy BestuurseenheidClassificatieCode:36a82ba0-7ff1-4697-a9dd-2e94df73b721 .
+    }
+}
+WHERE {
+    GRAPH ?g {
+        lblodRule:7fde1283-f580-4452-a5c4-f2234990d6e9 besluit:decidableBy BestuurseenheidClassificatieCode:36a82ba0-7ff1-4697-a9dd-2e94df73b721 .
+    }
+}
+
+;
+
+# Delete the NotificationRule from besluitType Budget
+DELETE {
+    GRAPH ?g {
+    BesluitType:40831a2c-771d-4b41-9720-0399998f1873 besluit:notificationRule ?rule.
+
+    ?rule ?p ?o .
+    }
+}
+WHERE {
+    GRAPH ?g {
+    BIND (lblodRule:13077265-a463-4690-b5f9-26729f2369d3 AS ?rule)
+    BesluitType:40831a2c-771d-4b41-9720-0399998f1873 besluit:notificationRule ?rule.
+    ?rule ?p ?o .
+    }
+}
+
+;
+
+# Delete duplicate prefLabel
+DELETE {
```

Comment:
wait how is this duplicate?

Reply:
When the label was adjusted, the previous label didn't get removed so the `BesluitDocumentType:18833df2-8c9e-4edd-87fd-b5c252337349` had two labels. Not really a duplicate indeed but it has two labels.